std/regex/internal/ir.d

```diff
@@ -971,6 +972,13 @@ struct CharMatcher {
         trie = makeTrie(set);
     }
 
+    immutable this(CodepointSet set)
+    {
+        auto asciiSet = set & unicode.ASCII;
+        ascii = BitTable(asciiSet);
+        trie = makeTrie(set);
```

Comment:
This trips the tester: `std/regex/internal/ir.d(979): Error: cannot implicitly convert expression `codepointSetTrie(((InversionList!(GcPolicy) __copytmp5388 = (__copytmp5388 = set).__fieldPostblit();) , __copytmp5388))` of type `Trie!(BitPacked!(bool, 1u), dchar, 1114112u, sliceBits!(8u, 21u), sliceBits!(0u, 8u))` to `immutable(Trie!(BitPacked!(bool, 1u), dchar, 1114112u, sliceBits!(8u, 21u), sliceBits!(0u, 8u)))`

Reply:
It’s all 32-bit platforms. This is strange the only thing that should change is ulong -> uint in the internal structure of a trie. Will see on the weekend if I can figure it out

Reply:
I don't know what I'm doing but I managed to sidestep this problem by passing CodepointSet by const ref.